CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## [1.1.0a1] - 2021-03-30
+
+### Changed
+
+- Dropped support for Python 2.7 and 3.5 ([#131](https://github.com/python-poetry/poetry-core/pull/131)).
```

Comment:
Might want to add a note for projects still needing this should use an appropriate build requires section.

Reply:
You're right. I just added a note to the change log.